geometry_msgs/Point current_state
float32 distance
---
geometry_msgs/Pose2D next_state